event_routing_backends/backends/events_router.py

```diff
@@ -78,29 +78,31 @@ def send(self, event):
         logger.debug('Successfully processed event %s for router with backend %s',
                      event, self.backend_name)
 
-        router = RouterConfiguration.get_enabled_router(self.backend_name)
+        routers = RouterConfiguration.get_enabled_routers(self.backend_name)
 
-        if not router:
+        if not routers or len(routers) == 0:
```

Comment:
this should be enough here
`if not routers`

Reply:
Done